### DISCUSSION TITLE
Why does Tesla Android need to be carried along in the car?

### DISCUSSION THREAD

**Discussion by RetoHB** [tesla-android/issue-tracker#171] [Ideas]
As I am not using Android Auto or CarPlay I am asking myself, why does Tesla Android need to be installed in the car? Why can't I leave it at home and access it over the standard Tesla Internet connectivity (of course password protected).

Is this something I can try myself with the current build? I think I would need the Tesla Android Webserver tell to also listen to connections on the Ethernet port and have it on DHCP. Then port forward external port 80 to my Pi4. Can this work?

**Comment by ochompsky** [comment]
i imagine it should work with a forwarded port.  performance will likely be horrific but im sure you know that.

**Reply by RetoHB** [reply]
You mean because of the slow Tesla Internet speed?

**Reply by mikegapinski** [reply]
Not only, Tesla Android transmits each frame as an image to bypass the video in motion restriction. If exposed externally you would get like 5fps tops + a terrible input lag
